charts/kube-otel-stack/values.yaml

```diff
@@ -115,11 +167,81 @@ tracesCollector:
+      # contains the OTel-Arrow exporter component.  When the image
+      # supports OTel-Arrow the "otlp" component name can be replaced
+      # with "otelarrow".  The non-Arrow configuration is identical
+      # between these components.
       otlp:
-        endpoint: ingest.lightstep.com:443
+        # OTel-Arrow notes: To use OTel-Arrow during early-access
+        # specifically requires the following endpoint.  This endpoint
+        # supports both OTLP and OTel-Arrow.
+        endpoint: spaningest.lightstep.com:443
         headers:
           "lightstep-access-token": "${LS_TOKEN}"
 
+        # OTel-Arrow notes:
+        #
+        # Queue settings are required.  It does not make sense to use
+        # the OTel-Arrow exporter without a queue, it has to do with
+        # requiring the "num_consumers" limit configured in this
+        # section (i.e., we require a queue in order to limit the
+        # number of concurrent exports).
+        #
+        # Note that the queue settings are applied in unit-terms
+        # produced by the batch processor, so a number like 100 means
+        # the queue has support for 100 pre-batched items.  With up to
+        # 1500 spans each (from the batch processor), this
+        # configuration allows 150,000 spans to occupy memory.
+        queue_settings:
+          enabled: true
+          num_consumers: 4
+          queue_size: 100
+
+        # Retry settings are optional.
+        #
+        # Note that while retries are attempted, this component will
+        # begin to drop arriving data if the queue is not large
+        # enough.
+        retry_on_failure:
+          # We recommend disabling retries, since while the export is
+          # blocked it is likely that arriving spans will drop, and
+          # Otherwise, collectors will need substantial additional
+          # memory to survive transient failures.  Nevertheless, we
+          # recommend a limited retry policy to gracefully occasional
+          # failures, paired with a modest queue size.
+          #
+          # Note there is a persistent storage option inherited from a
+          # common collector component.  When persistent storage is
+          # configured, the default retry configuration is sensible.
+          #
+          # For more details on retry and queue settings, please refer to
+          # https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md
+          enabled: true
+          max_elapsed_time: 60s
+
+        # While we expect latency under one second, typically, we
+        # recommend a longer timeout than the default.
+        timeout: 30s
+
+        # OTel-Arrow notes: these settings are specific to OTel-Arrow.
+        # To return this configuration to the standard OTLP exporter,
+        # replace "otelarrow" with "otlp" above and remove the "arrow"
+        # section below.
+        arrow:
```

Comment:
should we initially keep this commented out given it would cause a collector w/ contrib to fail?